test/integration/clusterresourceoverride_admission_test.go

```diff
@@ -60,6 +60,9 @@ func TestClusterResourceOverridePlugin(t *testing.T) {
 	}
 	_, err = limitHandler.Create(limit)
 	checkErr(t, err)
+	// stall in the hopes the LimitRanger cache will also get the new object before our next pod runs through it
+	_, err = limitHandler.Get(limit.Name)
```

Comment:
This won't do what we need and would only provide benefit based on the "sleep" affect it has.  We need to fix the limitrange admission plugin somehow.